components/aws/sagemaker/tests/integration_tests/utils/__init__.py

```diff
@@ -58,7 +58,7 @@ def run_command(cmd, *popenargs, **kwargs):
         pytest.fail(f"Command failed. Error code: {e.returncode}, Log: {e.output}")
 
 
-def read_from_file_in_tar(file_path, file_name, decode=True):
+def read_from_file_in_tar(file_path, file_name="data", decode=True):
```

Comment:
I did not follow where this `data` is coming from

Reply:
From experimentation, when using `outputPath` Argo stores it in a file called `data` inside the tarball.

Reply:
Could you add a comment here explaining this please.

Reply:
`data` comes from here: https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/_components.py#L182

It's an implementation detail of the path generation and is not guaranteed to be like that forever. (Although there are no plans to change it.)
In a sense, the .tgz archiving or the URL format are also implementation details.

Reply:
Thanks for that link. I will include it as part of the method comments.